### DISCUSSION TITLE
Support for MongoDB

### DISCUSSION THREAD

**Discussion by AndyTempel** [deta/cloud-docs#448] [General]
It's a bit weird to not support connections to external mongo databases. What's the exact reason behind this?

**Comment by jnsougata** [comment]
If you are talking about some persistent connection that's not possible cause you can think of micros as small tasks which runs for 10s and  after that they gets destroyed. that way it's not possible to keep the connection alive as micros are not actual VM

**Comment by abdelhai** [comment]
we have no issues with mongodb. mongo labs requires a static ip to allow servers to connect to it. as micros do now have a static ip, users use to report that mongodb is not working for them. i heard there might be a way around it but i'm not sure what it is. feel free to investigate. (as mentioned, this is not related to deta, rather it is/was an issue on mongdb's side)
